DNN Platform/Library/Entities/Modules/Settings/SettingsRepository.cs

```diff
@@ -228,6 +252,25 @@ private void DeserializeProperty(T settings, PropertyInfo property, ParameterAtt
                             Exceptions.LogException(exception);
                         }
                     }
+                } else if (propertyType.FullName == "System.TimeSpan")
```

Comment:
Why is this a string comparison, instead of `propertyType is TimeSpan` or `propertyType == typeof(TimeSpan)`?